Guide.docc/DataRaceSafety.md

```diff
@@ -60,11 +60,10 @@ see <doc:IncrementalAdoption#Dynamic-Isolation>
 ### Isolation Domains
 
 Data isolation is the _mechanism_ used to protect shared mutable state.
-But, it is often useful to talk about an independent unit of isolation.
+But it is often useful to talk about an independent unit of isolation.
 This is known as an _isolation domain_.
 How much state a particular domain is responsible for
-protecting can vary widely. An isolation domain might protect a single variable, 
-or an entire subsystem, like a complete user interface.
+protecting varies widely. An isolation domain might protect a single variable, or an entire subsystem, such as a user interface.
```

Comment:
```suggestion
protecting varies widely. An isolation domain might protect a single variable,
or an entire subsystem, such as a user interface.
```

Reply:
This is kinda silly, but this document uses semantic line breaks and strives for an 80 char limit and I *think* this line is too long.